src/components/swimlanes/swimLanes.tsx

```diff
@@ -45,6 +46,15 @@ const SwimLanes = ({ lanesWithNumItems }) => {
                 alt: "",
                 aspectRatio: "twoByOne",
                 src: imageURL(collection.image_id, "full", "288,", "0"),
+                // component: (
+                //   <Image
```

Comment:
I just wanted to test Next.js' `Image` component to compare against the DS component. The Next.js one is _suppose_ to be faster but I can't really tell.